ajax/libs/kotlin/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "kotlin",
+  "version": "1.1.2-4",
```

Comment:
Please add the latest stable version. I don't see `v1.1.2-4` published on both GitHub and NPM.